apis/v1alpha1/instrumentation_types.go

```diff
@@ -133,6 +133,18 @@ type Java struct {
 	// Resources describes the compute resource requirements.
 	// +optional
 	Resources corev1.ResourceRequirements `json:"resources,omitempty"`
+
+	// Extensions defines java specific extensions.
+	// +optional
+	Extensions *Extensions `json:"extensions,omitempty"`
```

Comment:
I would prefer if this would be a list.

Reply:
You are right; this makes it easier to integrate some third-party extensions.